utils/interfaces/_library/core.py

```diff
@@ -129,6 +131,11 @@ def add_appsec_validation(self, request=None, validator=None, legacy_validator=N
             )
         )
 
+    def add_appsec_iast_validation(self, request=None, validator=None, is_success_on_expiry=True):
```

Comment:
```
def expect_iast_vulnerabilities(self, request, 
            type=None,
            location_path=None,
            location_line=None,
            evidence=None,
            vulnarability_count=None)
```

if type/location_path/location_line/evidence is specified, the values will be checked

if vulnarability_count is not specified, we don't care about the vulnarability count that fullfil the requirement. If it specified, we check the vulnarability count that fullfil the requirements